server/watson-discovery-service.js

```diff
@@ -5,8 +5,8 @@ const vcapServices = require('vcap_services');
 const discoveryCredentials = vcapServices.getCredentials('discovery');
 
 const discovery = new DiscoveryV1({
-  username: discoveryCredentials.username,
-  password: discoveryCredentials.password,
+  username: process.env.DISCOVERY_SERVICE_USERNAME || discoveryCredentials.username,
```

Comment:
This worked correctly. vcapServices will find the creds in either the vcap vars or in .env